addons/project_todo/__manifest__.py

```diff
@@ -5,31 +5,31 @@
     'name': 'To-Do',
     'version': '1.0',
     'category': 'Productivity/To-Do',
-    'website': 'https://www.odoo.com/app/todo', # TODO: This page has to be created
+    'website': 'https://www.odoo.com/app/project_todo', # TODO: This page has to be created
```

Comment:
I think we could keep the old url you put since that module will represent `Todo` app and not `Project Todo`  🙂

Reply:
Just to not have any confusion, I am not against to rename todo module into project_todo as Thibault suggested. 🙂